packages/Teams/js/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@microsoft/bot-components-teams",
+  "author": "Microsoft Corp.",
```

Comment:
In the other packages, we've used the author 'Bot Framework'. This feels like the more correct author we ought to be using. Do we need to get this aligned on the others? I don't know if there's any team-wide guidance on this.

Reply:
I don't know how important this key is. In fact, it generally includes an email address. Perhaps something we can reconcile post GA.